_component_design/breadcrumbs.md

```diff
@@ -51,65 +79,126 @@ Breadcrumbs offer a way to articulate the platform architecture in a way the use
 
 <section class="static-section" markdown="1">
 
-## Standard breadcrumbs
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Composition
+
+Breadcrumbs contain the following elements:
+
+- **Root (required)**: The breadcrumb that links to the top level of the application.
+- **Parent (required, if applicable)**: The breadcrumb that links to the page one level up from the user's current location.
+- **Current (required)**: The breadcrumb that shows the user's current location.
+- **Delimiter (required)**: The delimiter that separates the breadcrumbs.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Breadcrumb composition"] [class:"image bg-light border"] %}
+<embed src="{{site.baseurl}}/assets/images/components/navigation/breadcrumbs/breadcrumbs-composition.png" width="318"/>
+
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
 
 <div class="hxRow"  markdown="1">
+
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Specifications
+### Style
 
-- __Placement:__ Keep breadcrumbs above the major heading of the page, 8px from the eyebrow. The container for breadcrumbs will always live on the top left corner of the page when it is present. Make sure the breadcrumb container is 16px above the content on the top of the page.
-- __Container & Spacing:__ The breadcrumbs container is 28px tall and full-page wide if it needs to be. We recommend shortening measures below. Breadcrumbs are in all caps with arrow icons in `gray 600` in-between.
-- __Alignment:__ Breadcrumbs should be left aligned to the left-most content on the page as noted in [Layout]({{site.baseurl}}/layout/page-layouts.html).
+Use these specifications when you construct breadcrumbs.
```

Comment:
need a link bubble ref to link in the visual

Reply:
Done.